client/src/app/pages/issues/issue-detail-drawer/file-incidents-detail-modal/file-all-incidents-table.tsx

```diff
@@ -143,3 +143,8 @@ export const FileAllIncidentsTable: React.FC<
     </>
   );
 };
+
+const getFirstNonEmptyLine = (message: string) => {
+  const lines = message.split("\n");
+  return lines.find((line) => line.trim() !== "") || "No content available.";
```

Comment:
IMHO the no-content label should be styled as a placeholder to clearly distinguish from the analyzed code. If not possible I would use empty string.

Reply:
Since this text is parsed as Markdown, there are some edge cases that _could_ be a problem.  If the first non-empty line is triple back-tick (like for a code block), weird things could happen.

I don't mind if there is markdown message here like `_(empty message)_` or some such, but it would make more sense up in the `<Td>`.